media-libs/openexr/metadata.xml

```diff
@@ -4,7 +4,13 @@
 	<maintainer type="project">
 		<email>media-video@gentoo.org</email>
 	</maintainer>
+	<use>
+		<flag name="utils">
+			Build several utility binaries for verifying and manipulating
+			EXR files.
```

Comment:
unnecessary line break.

Reply:
The break is a coloumn 74, which is already higher then 70. I don't treat this break unnecessary.

Reply:
70 cols is your personal maximum? ;) But fine, I'm not going to make it a blocker.

Reply:
no it's not my personal maximum, this is more in the range of 120, but it's the maximum of many things related to coding.

Reply:
From https://devmanual.gentoo.org/ebuild-writing/file-format/index.html: `Where possible, try to keep lines no wider than 80 positions.` Although this refers to ebuilds, I would also use this guideline for `metadata.xml`.

Reply:
I feel like this is not written with .xml files in mind, with all the indentation going on. But it is nothing I am obsessed about.